GLMakie/src/glwindow.jl

```diff
@@ -107,6 +107,11 @@ function GLFramebuffer(fb_size::NTuple{2, Int})
         internalformat = GL_DEPTH24_STENCIL8,
         format = GL_DEPTH_STENCIL
     )
+
+    # Holds the (temporary) render of a scene
+    color_buffer = Texture(
+        RGBA{N0f8}, fb_size, minfilter = :nearest, x_repeat = :clamp_to_edge
```

Comment:
Does this mean that GLMakie can support transparent backgrounds?  Or would that need improvements to the shader / OIT?

Reply:
In saved images with some more fixes yes, more or less

```julia
begin
    scene = Scene(backgroundcolor = (:orange, 0.3))
    s1 = Scene(scene, Rect2f(100, 100, 100, 100), backgroundcolor = (:cyan, 0.3), clear = true);
    s2 = Scene(scene, Rect2f(300, 100, 100, 100), backgroundcolor = (:purple, 0.3), clear = true);
    scatter!(scene, rand(10), rand(10))
    screen = display(scene)
end
```

![test](https://github.com/user-attachments/assets/3f75fa5e-6ab6-4651-876c-d90d9568eddf)

The main thing that's clashing with transparent backgrounds here is FXAA. It requires opaque pixels to do it's thing which you don't have in a transparent render. And also not in a transparent subscene.
Another annoyance is that you can't work with the usual `(c.a * c.rgb + (1-c.a) * background.rgb), 1)` blending because it assumes the background is opaque. I think with two transparent colors you have to do `(c1.a * c1.rgb + (1-c1.a) * c2.a * c2.rgb, (1-c1.a) * (1-c2.a))` and do the final background blend as `(composed.rgb + composed.a * background.rgb, 1)`.
But OIT doesn't care. Plot rendering doesn't care. SSAO I haven't tested but it also shouldn't care.
(Transparent windows I assume aren't possible from a GLFW/OS side but I haven't checked.)

Reply:
I see, I guess we could always turn FXAA off in the short term though.  

What I meant was having a completely transparent background - at which point I guess you don't need a final blend.  (Motivation was finally finishing that CairoMakie PR that allows rendering from other backends natively)

https://github.com/glfw/glfw/issues/197 seems to indicate that GLFW supports transparent framebuffers and windows via a window hint.